src/components/homesections/homesections.js

```diff
@@ -243,9 +243,11 @@ define(['connectionManager', 'cardBuilder', 'appSettings', 'dom', 'apphost', 'la
         return function (items) {
             var cardLayout = false;
             var shape;
-            if (itemType === 'Channel' || viewType === 'movies' || viewType === 'books') {
+            console.warn(viewType);
+            console.warn(itemType);
```

Comment:
:P

Reply:
Dammit :p

Reply:
I need to start using a debugger instead of always doing thing and forgetting to remove them x)